### PR TITLE
Crm 6.0.0 task

### PR DESCRIPTION
**Pull request by Bhimashankarsg**
addedd Bhimashankar_GGGGGGMMMMMMMMM

### CODE REVIEW

File: Automation_Framework_CRM_Task/src/main/java/com/CRM/ObjectRepository/ProductInfoPage.java [hunk 1]
```diff
@@ -12,7 +12,20 @@ public class ProductInfoPage {
 	public ProductInfoPage(WebDriver driver) {
 		PageFactory.initElements(driver, this);
 	}
+	// Bhimashankar_GGGGGGMMMMMMMMM
+		@FindBy(id = "dtlview_Website")
```
Comment (Bhimashankarsg) [inline Automation_Framework_CRM_Task/src/main/java/com/CRM/ObjectRepository/ProductInfoPage.java:16]: This line is modified

**Comment by Asha306** [comment]
dont use index

**Comment by Bhimashankarsg** [comment]
DONE

**Comment by Asha306** [comment]
merge successfully

**Pull request merged**